src/main/java/org/opentripplanner/middleware/persistence/Persistence.java

```diff
@@ -0,0 +1,82 @@
+package org.opentripplanner.middleware.persistence;
+
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoDatabase;
+import org.bson.codecs.configuration.CodecProvider;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.bson.codecs.pojo.PojoCodecProvider;
+import org.opentripplanner.middleware.models.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.bson.codecs.configuration.CodecRegistries.fromProviders;
+import static org.bson.codecs.configuration.CodecRegistries.fromRegistries;
+
+/**
+ * Groups together a bunch of TypedPersistence abstractions around MongoDB Collections.
+ */
+public class Persistence {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Persistence.class);
+    private static final String MONGO_URI = "MONGO_URI";
+    private static final String MONGO_DB_NAME = "MONGO_DB_NAME";
+
+    private static MongoClient mongo;
+    private static MongoDatabase mongoDatabase;
+    private static CodecRegistry pojoCodecRegistry;
+
+    // One abstracted Mongo collection for each class of persisted objects
+    public static TypedPersistence<User> users;
+
+    public static void initialize () {
+
+//        PojoCodecProvider pojoCodecProvider = PojoCodecProvider.builder()
+//            .register("com.conveyal.datatools.manager.jobs")
+//            .register("com.conveyal.datatools.manager.models")
+//            .register("com.conveyal.gtfs.loader")
+//            .register("com.conveyal.gtfs.validator")
+//            .automatic(true)
+//            .build();
```

Comment:
Since this is all copy-pasted anyways from elsewhere, I think we should leave these comments out for now.